WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/Preview/ProductDetailPreviewView.swift

```diff
@@ -0,0 +1,252 @@
+
+    init(viewModel: ProductDetailPreviewViewModel) {
+        self.viewModel = viewModel
+    }
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: Layout.blockVerticalSpacing) {
+                VStack(alignment: .leading, spacing: Layout.contentPadding) {
+                    // Title label.
+                    Text(Localization.title)
+                        .fontWeight(.bold)
+                        .titleStyle()
+
+                    // Subtitle label.
+                    Text(Localization.subtitle)
+                        .foregroundColor(.secondary)
+                        .bodyStyle()
+                }
+                .padding(.bottom, Layout.titleBlockBottomSpacing)
+
+                // Product name
+                VStack(alignment: .leading, spacing: Layout.contentVerticalSpacing) {
+                    Text(Localization.productName)
+                        .foregroundColor(.primary)
+                        .subheadlineStyle()
+                    BasicDetailRow(content: viewModel.generatedProduct?.name,
+                                   isLoading: viewModel.isGeneratingDetails)
+                }
+
+                // Product description
+                VStack(alignment: .leading, spacing: Layout.contentVerticalSpacing) {
+                    Text(Localization.productDescription)
+                        .foregroundColor(.primary)
+                        .subheadlineStyle()
+                    BasicDetailRow(content: viewModel.generatedProduct?.fullDescription,
+                                   isLoading: viewModel.isGeneratingDetails)
+                }
+
+                // TODO: update values based on product
+                // Other details
+                VStack(alignment: .leading, spacing: Layout.contentVerticalSpacing) {
+                    Text(Localization.details)
+                        .foregroundColor(.primary)
+                        .subheadlineStyle()
+                    TitleAndValueDetailRow(title: Localization.productType,
+                                           value: "Physical",
+                                           image: UIImage.productImage,
+                                           isLoading: viewModel.isGeneratingDetails,
+                                           cornerRadius: Layout.cornerRadius)
+                    .padding(.bottom, Layout.contentPadding)
+
+                    VStack(spacing: 0) {
+                        TitleAndValueDetailRow(title: Localization.price,
+                                               value: "Regular price: $15",
+                                               image: UIImage.priceImage,
+                                               isLoading: viewModel.isGeneratingDetails)
+                        Divider()
+                            .background(Color(.separator))
+                        TitleAndValueDetailRow(title: Localization.inventory,
```

Comment:
Opinion: Should inventory be part of the preview screen? I don't think we will generate anything related to stock status using AI.

Reply:
I see this on the design, my understanding is that we can add that extra information for the product ourselves as we expect the new product to be in stock. Does that make sense? I'll ping the team about that too.